templates/versafix-1/template-versafix-1.html

```diff
@@ -1831,8 +1986,7 @@
                            width="570" style="text-align:center;width:100%;max-width:570px;box-sizing:border-box;padding:0">
                         <tr>
                             <td align="center" valign="top" style="text-align:center;padding:20px 0 0 0;width:90%;max-width:530px;border-top:1px solid #e8e8e8;">
-                                <span style="font-family:'Helvetica',sans-serif;font-family: Arial, Helvetica, sans-serif;font-weight:normal;font-size:13px;color:#3f3f3f;opacity:.5;width:100%;text-align:center;display:block;
```

Comment:
why this change is here?
I made it the way that sponsor block font is controlled